components/src/core/hero/hero.component.ts

```diff
@@ -1,19 +1,22 @@
-import { Component, Input, OnInit } from '@angular/core';
+import {Component, Input, OnChanges} from '@angular/core';
+export type IconSize = 'normal' | 'large' | number;
+export type FontSize = 'small' | 'normal';
 
 @Component({
     selector: 'pxb-hero',
     template: `
-        <div class="wrapper">
-            <span
-                class="icon"
+        <div class="root">
+            <div
+                class="primary-icon"
+                [style.backgroundColor]="iconBackgroundColor"
```

Comment:
this backgroundColor doesn't have any padding around icons that are close to the edge of the 24x24 grid:
![image](https://user-images.githubusercontent.com/29152776/78071743-feb21500-736b-11ea-924d-896e6d400eee.png)

Reply:
I have to check to see if this isn't an issue in the React library too.

Reply:
![image](https://user-images.githubusercontent.com/6538289/78261324-181ea280-74cd-11ea-9974-99f4f6a32290.png)

This is from the react library.  If a user is trying to use an icon that requires additional padding, I think they can just add the padding themselves.  Most icons dont stretch all the way vertically or horizontally, so I think Cloud is in the minority here.

Reply:
It's not just cloud...it's pretty much any square-shaped icon too. But since it's also an issue in React, we don't have to deal with it right now.